src/Notify/composer.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "symfony/ux-notify",
+    "type": "symfony-bundle",
+    "description": "Native notification integration for Symfony",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathias Arlaud",
+            "email": "mathias.arlaud@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Notify\\": "."
+        },
+        "exclude-from-classmap": [
+            "/Tests/"
+        ]
+    },
+    "require": {
+        "php": ">=7.2.5",
```

Comment:
I think, for new stuff, we should require php 8.0 at least. But I've asked the core team to weigh in